files/pt-br/learn/html/multimedia_and_embedding/video_and_audio_content/index.html

```diff
@@ -220,7 +220,7 @@ <h2 id="Exibindo_trilhas_de_texto_em_vídeo">Exibindo trilhas de texto em vídeo
  <li>Da mesma forma, em ambientes em que a reprodução do áudio seria uma distração ou perturbação (como em uma biblioteca ou quando um parceiro está tentando dormir), ter legendas pode ser muito útil.</li>
 </ul>
 
-<p>Não seria bom poder fornecer a essas pessoas uma transcrição das palavras que estão sendo ditas no áudio / vídeo? Bem, graças ao vídeo HTML5, você pode, com o formato <a href="https://wiki.developer.mozilla.org/en-US/docs/Web/API/WebVTT_API">WebVTT </a>e o elemento {{htmlelement ("track")}}.</p>
+<p>Não seria bom poder fornecer a essas pessoas uma transcrição das palavras que estão sendo ditas no áudio / vídeo? Bem, graças ao vídeo HTML5, você pode, com o formato <a href="/en-US/docs/Web/API/WebVTT_API">WebVTT </a>e o elemento {{htmlelement ("track")}}.</p>
```

Comment:
Ideally, all links should be `pt-BR` actually. 
```suggestion
<p>Não seria bom poder fornecer a essas pessoas uma transcrição das palavras que estão sendo ditas no áudio / vídeo? Bem, graças ao vídeo HTML5, você pode, com o formato <a href="/pt-BR/docs/Web/API/WebVTT_API">WebVTT </a>e o elemento {{htmlelement ("track")}}.</p>
```

When Yari builds, it'll check that and if the slug `Web/API/WebVTT_API` hasn't been translated yet, it automatically turns the link into `<a href="/en-US/docs/Web/API/WebVTT_API">`

But it's more important to get rid of these old absolute URLs for now.

Reply:
@peterbe All right! I think I can help with that too 😉